src/common/gui/ThemeController.ts

```diff
@@ -70,6 +70,46 @@ export class ThemeController {
 		}
 	}
 
+	/**
+	 * Color token mapper from the old to the new.
+	 * If there are colors explicitly defined for the new tokens, they will be respected and the mapped colors will be overwritten.
+	 * Since the number of new tokens is less than the number of old tokens, it is impossible to map all colors. Thus, this is just for mitigation purposes.
+	 * This mapper could be removed after all users who have whitelabel color customization have migrated to the new color tokens.
+	 */
+	public static mapOldToNewColorTokens(customizations: Partial<ThemeCustomizations>): ThemeCustomizations {
```

Comment:
old customizations are not `Partial<ThemeCustomizations>` right? It is rather `keyof typeof oldToNewColorTokenMap`